compiler/stmt.py

```diff
@@ -286,6 +309,21 @@ def visit_ImportFrom(self, node):
                 mod.expr, self.block.intern(name))
             self.block.bind_var(
                 self.writer, alias.asname or alias.name, member.expr)
+    elif node.module == '__future__':
+      for alias in node.names:
+        name = alias.name
+        if name in future_features:
+          (flag, implemented) = future_features[name]
+          if not implemented:
+            msg = 'future feature {} not yet implemented by grumpy'.format(name)
+            raise util.ParseError(msg)
+          self.parser_flags |= flag
+        elif name in redundant_future_features:
+          # no-op
+          pass
+        else:
```

Comment:
Kill the no-op elif and replace it with:

elif name not in redundant_future_features:
   ... raise ParseError